examples/jest-snapshot-testing/src/components/App.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import AppHeader from './AppHeader';
+import AppMain from './AppMain';
+import AppFooter from './AppFooter';
+
+export default () => (
+  <div className="app">
+    <AppHeader/>
+    <AppMain/>
+    <AppFooter/>
+  </div>
+);
```

Comment:
Can you make sure the files end with a newline so that Git can track future changes better.